### PR TITLE
Portfolio: Desktop version

### PR DESCRIPTION
**Pull request by sanieni6**
## Portfolio: Desktop version
New features:
+ Use media queries for the desktop version
+ Apply new styles for the desktop version
+ Change some tags in the index files

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -623,3 +632,453 @@ button:disabled {
...
+    margin-top: 130px;
+  }
+
+  .about-languages {
+    align-items: flex-start;
+    padding: 0 12px;
+    gap: 8px;
+    width: 186px;
+    height: 216px;
+  }
```
Comment (franclobo) [inline styles.css:941]: - [ ] Please complete the list of technical skills, take in mind there are different skills in each one, your design should be similar to the Figma design 👇🏽 
- [ ] Kindly use the styles described in the Figma design especially the family fonts to be similar to the original design as much as possible.

| Figma design | Your design |
|---|---|
| image | image |

File: styles.css [hunk 2]
```diff
@@ -623,3 +632,453 @@ button:disabled {
...
+  .primary-text {
+    margin-top: 15.27vw;
+    width: 31.59vw;
+    height: max-content;
+    padding: 0;
+    font-size: 2.77vw;
+    line-height: 3.61vw;
+    color: #262626;
+    text-align: left;
+  }
```
Comment (franclobo) [inline styles.css:963]: - [ ] Please 🙏🏽  use the styles described in the Figma design, especially the font family.

image

File: styles.css [hunk 3]
```diff
@@ -623,3 +632,453 @@ button:disabled {
...
+  .card-works {
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    place-content: center;
+    justify-content: center;
+    row-gap: 37px;
+    column-gap: 24px;
+    height: max-content;
+    width: 80.2vw;
+  }
```
Comment (franclobo) [inline styles.css:796]: - [ ] Kindly correct the title of each card to be similar to the Figma design.

| Figma design | Your design |
|---|---|
| image | image |

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @sanieni6, 

# Required Changes ♻️

so-close-jim-halpert

Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sanieni6** [comment]
## Required changes done!!!
Changes done:
+ Complete the list of technical skills
+ Change the family fonts
+ Correct the title of each card

**Review by NahnahAJ — APPROVED** [review]
Hi @sanieni6 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Professional README :heavy_check_mark: 
- UI Design :heavy_check_mark: 
- No linter errors :heavy_check_mark: 

## Optional suggestions
-  [ ] I noticed that your project has a scroll effect. It might be because one of your elements is overflowing out of its container. I will suggest you fix this so your project won't act differently on different desktop screen sizes :100: I have attached a video for your reference :wink: 
https://www.loom.com/share/2726bd6f93d14d5fb9ce796373ebddd7


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**